common/enclave/generate_mrenclave.sh

```diff
@@ -19,7 +19,6 @@ go_name=mrenclave.go
 cd $build_dir
 sgx_sign gendata -enclave enclave.so -config $enclave_dir/enclave.config.xml -out $tmp_name
 dd if=$tmp_name bs=1 skip=188 of=$hex_out_name count=32
-hexdump -C $hex_out_name
```

Comment:
Removed as it only outputs the hexdata here but nothing more. Also hexdump was not available in new dockerbase image and decided not to add it.

Reply:
we might want at some point remove it completely and just rely on the -dumpfile option of sgx_sign (see issue #77)